bucket_map/src/bucket_storage.rs

```diff
@@ -133,7 +136,7 @@ impl BucketStorage {
         }
     }
 
-    pub fn allocate(&self, ix: u64, uid: Uid) -> Result<(), BucketStorageError> {
+    pub fn allocate(&self, ix: u64, uid: Uid, resizing: bool) -> Result<(), BucketStorageError> {
```

Comment:
Minor: How do you feel about `is_resizing` here instead?

Also, I'd find it useful to have a doc comment on this fn that describes what the `is_resizing` flag is used for. Would you be willing to add that here?